src/apexmodels/apexModel.js

```diff
@@ -1,6 +1,9 @@
+const StringUtils = require('../utils/stringUtils');
```

Comment:
Within this repo we're using the file extension in the require path. It's a mild stylistic issue and we don't have a style guide.